packages/opa-react/src/use-authz.ts

```diff
@@ -25,76 +25,28 @@ export default function useAuthz(
   if (context === undefined) {
     throw Error("Authz/useAuthz can only be used inside an AuthzProvider");
   }
-  const { sdk, defaultPath, defaultInput, defaultFromResult } = context;
-
-  const [isLoading, setIsLoading] = useState(true);
-  const [result, setResult] = useState<Result>();
-  const [error, setError] = useState<Error>();
-
-  // TODO(sr): This feels wrong. Figure out just how wrong it is.
-  const requestMemo = useDeepCompareMemoize({
-    defaultPath,
-    defaultInput,
-    defaultFromResult,
-    input,
-    path,
-    fromResult,
-  });
-
-  const evaluate = useCallback<(signal: AbortSignal) => Promise<Result>>(
-    async (signal: AbortSignal): Promise<Result> => {
-      const {
-        defaultPath,
-        defaultInput,
-        defaultFromResult,
-        input,
-        path,
-        fromResult: fromR,
-      } = requestMemo;
-      const p = path ?? defaultPath;
-      const i = mergeInput(input, defaultInput);
-      const fromResult = fromR ?? defaultFromResult;
-      const fetchOptions = { signal };
-      return p
-        ? sdk.evaluate(p, i, { fetchOptions, fromResult })
-        : sdk.evaluateDefault(i, { fetchOptions, fromResult });
+  const { defaultPath, defaultInput, defaultFromResult, queryClient } = context;
+  const p = path ?? defaultPath;
+  const i = mergeInput(input, defaultInput);
+  const fromR = fromResult ?? defaultFromResult;
```

Comment:
I'd avoid these "shortcut" const names since the proper names are already taken from the args; just don't destruct the args

```
export default function useAuthz(props: UseAuthzProps) {
...
const path = props.path ?? defaultPath
...
```

Reply:
I'll take the advice, but in this case, it's three arguments to a function, not component props. I don't think I can combine them -- short of using `arguments`, or changing the function signature to take an object instead.